x/session/module/module.go

```diff
@@ -151,12 +149,18 @@ func (am AppModule) BeginBlock(_ context.Context) error {
 
 // EndBlock contains the logic that is automatically triggered at the end of each block.
 // The end block implementation is optional.
-func (am AppModule) EndBlock(ctx context.Context) error {
-	// Store the block hash at the end of every block, so we can query the block hash
-	// to construct the SessionID.
+// TODO_TECHDEBT( @red-0ne): Add unit/integration tests for this.
+func (am AppModule) EndBlock(goCtx context.Context) error {
+	logger := am.keeper.Logger().With("EndBlock", "SessionModuleEndBlock")
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	blockHeight := ctx.BlockHeight()
```

Comment:
I think we've more consistently called the "go context" `ctx` and the "sdk context" `sdkCtx`. IIRC, we made this switch during the v0.50 migration. The rationale being that `ctx` usage is more conventional. I believe that this is also consistent with the v0.50 usages that we observed.

```suggestion
func (am AppModule) EndBlock(ctx context.Context) error {
	logger := am.keeper.Logger().With("EndBlock", "SessionModuleEndBlock")
	sdkCtx := sdk.UnwrapSDKContext(ctx)
	blockHeight := sdkCtx.BlockHeight()
```